bench/bench.ml

```diff
@@ -90,32 +90,129 @@ let prepare_workspace () =
       Format.eprintf "cloning %s/%s@." pkg.org pkg.name;
       Package.clone pkg)
 
-let dune_build () =
+(* TODO move somewhere - can't go in Dune_util.Gc due to cycle with Dune_lang *)
+let gc_stat_decoder =
+  let open Dune_lang.Decoder in
+  enter @@ fields
+  @@ let* minor_words = field "minor_words" float in
```

Comment:
Instead of writing `let* foo = bar in let* baz = qux ...` you can write `let+ foo bar and+ baz = qux...` (this is true for all monadic modules not just `Dune_lang.Decoder`). This will result in slightly more efficient code and also match the style that most of code is written.